README.md

```diff
@@ -78,8 +77,29 @@ And the installed version of python will be printed.
 
 ---
 
+<details><summary>Step 3 : Installing Git </summary>
 
-<details><summary>Step 3 : Creating Project Directory </summary>
+Click on [Download Git](https://git-scm.com/downloads)
+
+</details>
+
+
+---
```

Comment:
Added Link to download the Git to maintain inheritance as readme contains links to download the other prerequisites.